packages/faust-nx/src/components/WordPressTemplate.tsx

```diff
@@ -2,40 +2,48 @@ import React, { PropsWithChildren } from 'react';
 import { DocumentNode, useQuery } from '@apollo/client';
 import { getTemplate } from '../getTemplate.js';
 import { SeedNode } from '../queries/seedQuery.js';
+import { WordPressTemplate } from '../getWordPressProps.js';
 import { getConfig } from '../config/index.js';
+import { usePreviewQuery } from '../hooks/usePreviewQuery.js';
 
 export type WordPressTemplateProps = PropsWithChildren<{
   __SEED_NODE__: SeedNode;
+  context?: { asPreview?: boolean };
```

Comment:
Since this will be a prop passed down to the app's `wp-templates`, we should name it something more specific to avoid collision. Maybe somthing like `__FAUST_CONTEXT__`